### ISSUE TITLE
<Trans ns="namespace">...</Trans> and t("...", { ns: "namespace" })

### ISSUE THREAD

**Issue by externuz** [i18next/i18next-parser#192]
- Is this a bug or a feature request?
bug
- What version are you running (`0.x` or `1.x`)? 
1.0.0-beta37
- Which runner (cli, gulp...)?
cli

## What I'm trying to do?

Im trying to parse .tsx file with such content:
```
      <div>
        <Trans ns="second">Welcome to react</Trans>
        <h1>{t("Welcome to React", { ns: "first" })}</h1>
      </div>
```

Here is my config file:

```
module.exports = {
  defaultNamespace: "translation",
  lexers: {
    ts: [{ lexer: "JavascriptLexer" }],
    tsx: [{ lexer: "JsxLexer" }],
    default: ["JavascriptLexer"]
  },
  input: ["src/**/*.{ts,tsx}"],
  locales: ["en", "ru"],
  output: "src/locales/$LOCALE/$NAMESPACE.json",
  useKeysAsDefaultValue: true,
  verbose: true
};
```

The command I'm running:

```
i18next
```

## What was the result?

`src/locales/$LOCALE/translation.json` files

## What was I expecting?

`src/locales/$LOCALE/first.json` and
`src/locales/$LOCALE/second.json`

**Comment by vincaslt** [comment]
Namespaces in `Trans` component might not be supported or might be broken, will have to check.
And the second way you used it, I don't think is possible in i18next itself, could you give a link to the docs for it? I checked and couldn't find. The only way I saw is `i18next.t('namespace:myKey')`, which should be supported by this parser.

**Comment by externuz** [comment]
`t` - TFunction
```
export interface TFunction {
  <
    TResult extends TFunctionResult = string,
    TKeys extends TFunctionKeys = string,
    TInterpolationMap extends object = StringMap
  >(
    key: TKeys | TKeys[],
    options?: TOptions<TInterpolationMap> | string,
  ): TResult;
  <
    TResult extends TFunctionResult = string,
    TKeys extends TFunctionKeys = string,
    TInterpolationMap extends object = StringMap
  >(
    key: TKeys | TKeys[],
    defaultValue?: string,
    options?: TOptions<TInterpolationMap> | string,
  ): TResult;
}
```

TOptions
```
export type TOptions<TInterpolationMap extends object = StringMap> = TOptionsBase &
  TInterpolationMap;
```

TOptionsBase
```
export interface TOptionsBase {
  /**
   * Default value to return if a translation was not found
   */
  defaultValue?: any;
  /**
   * Count value used for plurals
   */
  count?: number;
  /**
   * Used for contexts (eg. male\female)
   */
  context?: any;
  /**
   * Object with vars for interpolation - or put them directly in options
   */
  replace?: any;
  /**
   * Override language to use
   */
  lng?: string;
  /**
   * Override languages to use
   */
  lngs?: string[];
  /**
   * Override language to lookup key if not found see fallbacks for details
   */
  fallbackLng?: FallbackLng;
  /**
   * Override namespaces (string or array)
   */
  ns?: string | string[];
  /**
   * Override char to separate keys
   */
  keySeparator?: string;
  /**
   * Override char to split namespace from key
   */
  nsSeparator?: string;
  /**
   * Accessing an object not a translation string (can be set globally too)
   */
  returnObjects?: boolean;
  /**
   * Char, eg. '\n' that arrays will be joined by (can be set globally too)
   */
  joinArrays?: string;
  /**
   * String or array of postProcessors to apply see interval plurals as a sample
   */
  postProcess?: string | string[];
  /**
   * Override interpolation options
   */
  interpolation?: InterpolationOptions;
}
```

**Comment by heivo** [comment]
I created 2 PR's to address those namespace issues.

**Comment by karellm** [comment]
Thanks @heivo for the good work! I released `1.0.0-beta38` with your changes.
